i#793 license help

srerickson opened the pull request:
Adds a "read more" link to the license field hint text. Also includes some changes to the labels for available licenses.

cam156 commented on spec/models/user_mailer_spec.rb:
```diff
@@ -23,7 +23,7 @@
     let(:mock_service) { double }
     let(:csv)          { "a,b,c\nd,e,f\n" }
-    let!(:generic_work) { create :work, :with_pdf }
+    let!(:generic_work) { create :work }
```
@awead you are removing with pdf here.  Is there a reason?

awead replied:
@cam156 Whether or not the work has a pdf is irrelevant to the test. Omitting it has no effect on the outcome.

cam156 commented on spec/factories/file_sets.rb:
```diff
@@ -35,13 +35,6 @@
       end
     end
 
-    trait :pdf do
```
This was used until you removed it.

awead replied:
@cam156 If the `:with_pdf` work trait isn't needed, then this trait isn't needed either.

srerickson committed:
'Choose a license' markdown page

srerickson committed:
Updates to available licenses:
- change wording for CC license options
- change "rights reserved" option to "In Copyright (Rights Reserved)"
- add "No Copyright" option
- remove "public domain mark" option

srerickson committed:
fix formatting

srerickson committed:
fix no-copyright term #1103

rschenk committed:
Markdown support for field hints' translations (#1172)

whereismyjetpack committed:
Update ci (#1173)

* updates CI

* re-add redis check

whereismyjetpack committed:
updates rails (#1171)

whereismyjetpack committed:
fix ci (#1174)

nicktiberi committed:
Hide the "Done" button in the file upload component (#1176)

Per the Uppy Dashboard plugin docs, setting the doneButtonHandler
to null hides the Done button. It wasn't doing anything useful,
so it seems to make sense just to hide it.

whereismyjetpack committed:
2022 (#1179)

dependabot[bot] committed:
Bump nanoid from 3.1.23 to 3.2.0 (#1182)

Bumps [nanoid](https://github.com/ai/nanoid) from 3.1.23 to 3.2.0.
- [Release notes](https://github.com/ai/nanoid/releases)
- [Changelog](https://github.com/ai/nanoid/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ai/nanoid/compare/3.1.23...3.2.0)

---
updated-dependencies:
- dependency-name: nanoid
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

whereismyjetpack committed:
updates ruby, node, and bundler (#1181)

Commit:
UI for withdraw work versions (#1177)

* withdraw work versions admin work settings
* warning text for the version withdrawing

Co-authored-by: Ryan Schenk <rschenk@gmail.com>

rschenk committed:
Work factory keeps the same list of actors for all versions (#1178)

ajkiessl committed:
Limiting the dois returned from AllDois for WorkVersions (#1180)

rschenk committed:
Fix flaky specs from VCR and WebMock

rschenk committed:
AnalyticsController no longer raises ActiveRecord::RecordNotFound

rschenk committed:
Update default license (#1188)

srerickson committed:
fix spec

srerickson committed:
Revert "'Choose a license' markdown page"

This reverts commit 1f664b6a750855bb74e689dd003432cd8e710d23.

srerickson committed:
read more links for license hint text. #793

rschenk reviewed (commented):
I think this looks good, but it's really had to figure out what changes you actually made. If not too much of a hassle, can you please rebase your changes on top of develop rather than this merge?

srerickson commented:
I thought I did rebase -- some of these commits are really old so the
history is kind of a mess, especially given how minor the changes are. I
think I'll just make a new branch entirely. Sorry

On Wed, Jan 26, 2022 at 6:57 AM Ryan Schenk ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> I think this looks good, but it's really had to figure out what changes
> you actually made. If not too much of a hassle, can you please rebase your
> changes on top of develop rather than this merge?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/psu-libraries/scholarsphere/pull/1191#pullrequestreview-863573408>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGV6IYVLM4XN22VQLUB2G3UX74URANCNFSM5MZ2XTJA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

srerickson commented:
redoing this in a separate branch/pr
